src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -69,7 +74,15 @@ private class Indexing(resources: Resources[Task], cache: Caches[Task], coordina
           // format: off
           val project = Project(config.http.projectsIri + label, label, orgLabel, desc, base, vocab, am, uuid, orgUuid, 1L, deprecated = false, instant, subject.id, instant, subject.id)
           // format: on
-          cache.project.replace(project) *> coordinator.start(project)
+          implicit val s         = subject
+          val elasticView: View  = ElasticView.default(project.ref)
+          val sparqlView: View   = SparqlView.default(project.ref)
+          val resolver: Resolver = InProjectResolver.default(project.ref)
+          cache.project.replace(project) *>
```

Comment:
can we wrap this block in a `RetryStrategy[F]`? otherwise if one operation fails the coordinator won't start...